central/src/lib/API/gametemplates.ts

```diff
@@ -50,10 +51,23 @@ export const deleteGameTemplate = async (id: string): Promise<IGameTemplate | nu
   return null;
 };
 
-export const listGameTemplates = async (queryLimit: number, nextToken: string | null): Promise<{ gameTemplates: IGameTemplate[], nextToken: string } | null> => {
+export const listGameTemplates = async (listQuerySettings: IListQuerySettings | null): Promise<{ gameTemplates: IGameTemplate[], nextToken: string } | null> => {
   try {
-    const games = await gameTemplateAPIClient.listGameTemplates(queryLimit, nextToken);
-    return games;
+    const nextToken = listQuerySettings?.nextToken ?? null;
```

Comment:
breaks the query settings object apart to be used with APIClient queries based on type